package-overrides/npm/element-resize-detector@1.1.0.json

```diff
@@ -0,0 +1,9 @@
+{
+  "main": "dist/element-resize-detector",
+  "format": "cjs",
+  "shim": {
+    "dist/element-resize-detector": {
+      "exports": "utils"
+    }
+  }
```

Comment:
yeah, during bundling it says it cannot resolve that utils, all the other exports are fine though.